sdk/webpubsub/azure-messaging-webpubsub/src/main/java/com/azure/messaging/webpubsub/WebPubSubServiceClient.java

```diff
@@ -62,13 +62,12 @@ public WebPubSubClientAccessToken getClientAccessToken(GetClientAccessTokenOptio
             if (options.getExpiresAfter() != null) {
                 requestOptions.addQueryParam("minutesToExpire", String.valueOf(options.getExpiresAfter().toMinutes()));
             }
-            if (CoreUtils.isNullOrEmpty(options.getRoles())) {
+            if (!CoreUtils.isNullOrEmpty(options.getRoles())) {
```

Comment:
Thanks for fixing this. Can you also add some tests to ensure these cases are covered?

Reply:
Tests are added.